delta_go/check_submission.py

```diff
@@ -0,0 +1,175 @@
+import datetime
+import hashlib
+import sys
+import time
+import warnings
+from collections import defaultdict
+from pathlib import Path
+from typing import Any, Callable, Dict, Optional, Set, Tuple, Type, Union
+
+import numpy as np
+
+from game_mechanics import BOARD_SIZE, GoEnv, choose_move_randomly, load_pkl
+from torch import nn
+
+
+def check_submission(team_name: str, choose_move_no_network: Callable) -> None:
+    example_state, _, _, info = GoEnv(choose_move_randomly).reset()
+    expected_choose_move_return_type = (int, np.int64)
+    expected_pkl_output_type = Any
+    pkl_file = load_pkl(team_name)
+
+    max_time = 5
+    t1 = time.time()
+    choose_move_no_network(
+        example_state,
+    )
+    t2 = time.time()
```

Comment:
Isn't this file out of date?

Reply:
Plus, we want to check that we can initialise their `MCTS` class here too, right? So maybe add that to `delta-utils`?